web/src/elements/Base.ts

```diff
@@ -77,7 +77,8 @@ export class AKElement extends LitElement {
     }
 
     async getTheme(): Promise<UiThemeEnum> {
-        return rootInterface()?.getTheme() || UiThemeEnum.Automatic;
+        // return rootInterface()?.getTheme() || UiThemeEnum.Automatic;
+        return rootInterface()?.getTheme() || UiThemeEnum.Light;
```

Comment:
Disabling `Automatic` here means the listener doesn't get installed. The browser is being ignored.  Only the user's explicit setting, retrieved with `getTheme`, will be honored; if it's not there, the default is `Light`.